### DISCUSSION TITLE
[Question] When does my subscription start?

### DISCUSSION THREAD

**Discussion by Bittu5134** [community/community#77670] [GitHub Education] [answered]
### Hi everyone,

So I recently got the GitHub student developer pack and have a little question about the pack.

The benefits of the pack include a 3-6 month subscription on partner websites. So when does the subscription start? Just after I get the Developer pack or when I register on the website? What if I am already registered on the website?

Thank you in advance for helping

**Comment by mesh-dell** [comment, marked as answer]
The subscription for partner websites included in the GitHub Student Developer Pack typically starts when you register on the website and activate the benefit. If you are already registered on the website, you may need to link your GitHub account to the partner website to activate the benefit.

**Comment by ufuomasamson** [comment]
My free trial just finished and am finiding it hard to activate for the copilot pro plan and continue using it
